apps/finance/app/src/lib/token-utils.js

```diff
@@ -18,21 +19,22 @@ const PRESET_TOKENS = new Map([
       ETHER_TOKEN_FAKE_ADDRESS,
       ANT_MAINNET_TOKEN_ADDRESS,
       DAI_MAINNET_TOKEN_ADDRESS,
+      SAI_MAINNET_TOKEN_ADDRESS,
     ],
   ],
 ])
 
 // Some known tokens don’t strictly follow ERC-20 and it would be difficult to
-// adapt to every situation. The data listed in this map is used as a fallback
+// adapt to every situation. The data listed in this map is used as an override
 // if either some part of their interface doesn't conform to a standard we
 // support.
-const KNOWN_TOKENS_FALLBACK = new Map([
+const KNOWN_TOKENS_OVERRIDE = new Map([
   [
     'main',
     new Map([
       [
-        DAI_MAINNET_TOKEN_ADDRESS,
-        { symbol: 'DAI', name: 'Dai Stablecoin v1.0', decimals: '18' },
+        SAI_MAINNET_TOKEN_ADDRESS,
+        { symbol: 'SAI', name: 'Sai Stablecoin v1.0', decimals: '18' },
```

Comment:
Good to see they fixed that in MCD DAI!

Reply:
Yes, thankfully. I still don't know how they got to that bytes32 abomination.